unity-ggjj/Assets/Tests/PlayModeTests/Scenes/CreditsTests.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using Tests.PlayModeTests.Tools;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+
+namespace Tests.PlayModeTests.Scenes
+{
+    public class CreditsTests
+    {
+        [UnityTest]
+        public IEnumerator CreditsCanBeLoadedViaAction()
+        {
+            yield return SceneManager.LoadSceneAsync("Game");
+            var gameState = Object.FindObjectOfType<NarrativeGameState>();
+            var actionDecoder = Object.FindObjectOfType<ActionDecoderComponent>();
+            actionDecoder.Decoder.NarrativeGameState = gameState;
+            actionDecoder.OnNewActionLine("&LOAD_SCENE:Credits\n");
```

Comment:
Nit: Just to verify the SceneLoader actually doing its job, I would suggest asserting, that we're not actually starting in the `Credits` scene, and only are there, after the line has been parsed.
```suggestion
            Assert.AreNotEqual(SceneManager.GetActiveScene().name, "Credits");
            actionDecoder.OnNewActionLine("&LOAD_SCENE:Credits\n");
```

That all aside: It's so rare to see simple tests these days, so this is super well written. 👍

Reply:
I would assume SceneManager.LoadSceneAsync("Game") would either load the Game scene or give an error, but I've added your suggestion as a sanity check (as we're clearly not above those going by my other PR :P)